quinn/examples/single_socket.rs

```diff
@@ -0,0 +1,98 @@
+//! You can have multiple QUIC connections over a single UDP socket. This is especially
+//! useful, if you are building a peer-to-peer system where you potentially need to communicate with
+//! thousands of peers or if you have a
+//! [hole punched](https://en.wikipedia.org/wiki/UDP_hole_punching) UDP socket.
+//! In addition, both QUIC server and client can operate on the same UDP socket.
```

Comment:
```suggestion
//! In addition, QUIC servers and clients can both operate on the same UDP socket.
```

Reply:
`insecure_connection` example does not use this function and when it includes `common` module the compiler complains about unused function.